test/parallel/test-process-load-env-file.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
```

Comment:
can you add a test when the env-file overrides a valid ENV?

for instance: `TEST=1 node --env-file=.env test.js`

where `.env` contains: TEST=2